sormas-backend/src/main/java/de/symeda/sormas/backend/util/DtoHelper.java

```diff
@@ -204,7 +207,7 @@ public static <T extends AbstractDomainObject> T fillOrBuildEntity(EntityDto sou
 				target.setUuid(uuid);
 			} else if (DataHelper.isNullOrEmpty(source.getUuid())) {
 				// target has a uuid. do nothing -> gracefully handle missing uuids of children
-			} else if (!target.getUuid().equals(source.getUuid())) {
+			} else if (!target.getUuid().equals(source.getUuid()) && !allowUuidOverwrite) {
```

Comment:
```suggestion
			} else if (!target.getUuid().equals(source.getUuid())) {
				if (allowUuidOverwrite)  {
					target.setUuid(source.getUuid());
				} else {			
```

Reply:
I think we have to set the uuid, right?

Reply:
indeed, don't know what was in my head :). I updated the logic.